src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -55,13 +55,14 @@ public enum Flags
         private SourceList _before;
         private SourceList _after;
         private readonly Features _features;
+        private readonly ParserOptions _parserOptions;
 
         // Represents temporary extra trivia, for when a parsing method
         // had to parse tailing trivia to do 1-lookahead. Will be
         // collected by the next call to ParseTrivia.
         private ITexlSource _extraTrivia;
 
-        private TexlParser(IReadOnlyList<Token> tokens, Flags flags, Features features = null)
+        private TexlParser(IReadOnlyList<Token> tokens, Flags flags, ParserOptions parserOptions, Features features = null)

```

Comment:
> Flags flags, ParserOptions parserOptions [](http://example.com/codeflow?start=55&length=41)

We have two ways now which could be conflicting. Can we have a single way of defining the behavior? Can we add a flag as well for this?